servant-client/test/Servant/ClientSpec.hs

```diff
@@ -155,6 +158,7 @@ server = serve api (
        return alice
   :<|> return NoContent
   :<|> (\ name -> return $ Person name 0)
+  :<|> (\ (name : _) -> return $ Person name 0)
```

Comment:
Why not use the whole list? E.g. `\names -> return $ zipWith Person names [0..]`.